ietf-layer0-types-ext.yang

```diff
@@ -568,6 +568,31 @@ module ietf-layer0-types-ext {
     }
   }  
 
+  grouping pdl-penalty {
+    description "entries of table; pair of values
+    polarization dependent loss and
+    associated penalty";
+
+    leaf max-polarization-dependent-loss {
+      type decimal64 {
+        fraction-digits 2;
+      }
+      units dB ;
+      description
+        "Maximum acceptable accumulate polarization dependent loss";
+    }
+
+    leaf penalty {
+      type decimal64 {
+        fraction-digits 2;
+        range "0..max";
+      }

```

Comment:
From the initial proposal I thought it could be an unsigned integer (uint8 may be sufficient) to encode 0, 2, 4 or 5 dB.

https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/issues/41#issue-723272062

Reply:
```suggestion
      type uint8;
```